cmd/glasskube/cmd/install.go

```diff
@@ -61,6 +62,34 @@ var installCmd = &cobra.Command{
 			cancel()
 		}
 
+		// if --no-wait is used
+		if installCmdOptions.NoWait {
+			go func() {
+				status, err := install.NewInstaller(client).
+					WithStatusWriter(statuswriter.Spinner()).
+					InstallBlocking(ctx, packageName, installCmdOptions.Version)
+
+				if err != nil {
+					fmt.Fprintf(os.Stderr, "An error occurred during installation:\n\n%v\n", err)
+					os.Exit(1)
+				}
+				if status != nil {
+					switch status.Status {
+					case string(condition.Ready):
+						fmt.Printf("✅ %v is now installed in %v.\n", packageName, config.CurrentContext)
+					default:
+						fmt.Printf("❌ %v installation has status %v, reason: %v\nMessage: %v\n",
+							packageName, status.Status, status.Reason, status.Message)
+					}
+				} else {
+					fmt.Fprintln(os.Stderr, "Installation status unknown - no error and no status have been observed (this is a bug).")
+					os.Exit(1)
+				}
```

Comment:
The non-blocking `Install` action (see other comment) does not return a status value, obviously, but we don't want to handle that anyways  if --no-wait has been used in the call.